Docs: Add docs for [plugin.plugin_id].tracing config key (#65900)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/_index.md

<hr>

> ## [plugin.plugin_id]

Reviewer: maybe it's better to move this before `## [plugin.grafana-image-renderer]` (line 2023) to have all the [plugin.*] related sections close together

> This section allows to configure plugin-specific settings. `plugin_id` must be replaced with the plugin id present in `plugin.json`.

Reviewer: Maybe add something along the lines "properties described in this section are available for all plugins but it should be set independently"

### tracing

> **Note**: Available in Grafana v9.5.0 or later, and [OpenTelemetry must be configured as well](#tracingopentelemetry).

> If `true`, propagate the tracing context to this backend plugin and enable tracing in the plugin (if it supports it).

Reviewer: ```suggestion
If `true`, propagate the tracing context to the plugin backend and enable tracing (if it supports it).
```

<hr>

## [plugin.plugin_id]

> This section allows to configure plugin-specific settings. `plugin_id` must be replaced with the plugin id present in `plugin.json`.

Reviewer 2: ```suggestion
This section explains how to configure plugin-specific settings. Replace thge `plugin_id` attribute with the plugin ID present in `plugin.json`.
```

Author: I have committed the second part ("must be replaced" -> "replace the"), but I am not sure about the first one.

I was referring to the `[plugin.plugin_id]` ini file section, not the paragraph.

Do you think it's okay to leave it as "This section allows to configure plugin-specific settings" or maybe "This section can be used to configure plugin-specific settings" ?

Thanks 😄

Reviewer 2: Okay, I misunderstood the sentence. You might want to reword to "This section can be used..." or "This section of the of the ini file..."

> Properties described in this section are available for all plugins, but must be set for each plugin.

Reviewer 2: ```suggestion
Properties described in this section are available for all plugins, but you must set them individually for each plugin.
```

### tracing

> **Note**: Available in Grafana v9.5.0 or later, and [OpenTelemetry must be configured as well](#tracingopentelemetry).

> If `true`, propagate the tracing context to the plugin backend and enable tracing (if it supports it).

Reviewer 2: ```suggestion
If `true`, propagate the tracing context to the plugin backend and enable tracing (if the backend supports it).
```
